inputfiles/addedTypes.json

```diff
@@ -1765,5 +1765,568 @@
+                "name": "DRAW_BUFFER0_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER1_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER2_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER3_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER4_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER5_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER6_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER7_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER8_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER9_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER10_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER11_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER12_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER13_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER14_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "DRAW_BUFFER15_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "MAX_COLOR_ATTACHMENTS_WEBGL",
+                "type": "number"
+            },
+            {
+                "readonly": true,
+                "name": "MAX_DRAW_BUFFERS_WEBGL",
+                "type": "number"
+            }
+        ],
+        "methods": [
+            {
+                "name": "drawBuffersWEBGL",
+                "signatures": [
+                    "drawBuffersWEBGL(buffers: number[]): void"
+                ]
+            }
+        ]
+    },
+    {
+        "kind": "interface",
+        "name": "WEBGL_lose_context",
+        "flavor": "DOM",
+        "constructorSignatures": [
```

Comment:
i do not think you need these. these are types only and do not exisit at runtime.

Reply:
Only `constructorSignatures` should be removed.